libs/app-theme/ui/src/lib/icon/icons.ts

```diff
@@ -33,7 +33,7 @@ export const icons = {
   [IconType.Backup]: Backup,
   [IconType.ChevronDown]: ChevronDown,
   [IconType.Close]: Close,
-  [IconType.Contacts]: Contacts,
+  [IconType.ContactsBook]: Contacts,
```

Comment:
inconsistency 😉

Reply:
Fixed in [6dff0f5](https://github.com/mudita/mudita-center/pull/2460/commits/6dff0f56533a2eec13cebf52589fb06a823b3dcc)